### PR TITLE
Feature/change name field

### PR DESCRIPTION
**Pull request by nmenezes0**
This PR does the following:

1. Allows you to pass in the display value for choices fields (levels for skills and languages) ie "Basic" instead of "basic" etc
2. Returns the ID of skills/languages/skills to develop
3. Changes `skill_name` to `name` for skills and skills to develop
4. Updates relevant fields in the admin

@rottitime - for info.

### CODE REVIEW

File: api/ellandi/registration/serializers.py [hunk 1]
```diff
@@ -19,6 +19,27 @@
 )
 
 
+class ChoiceDisplayField(serializers.ChoiceField):
+    """
+    Convert from display value to internal valuefor choice fields.
```
Comment (Singletoned) [inline api/ellandi/registration/serializers.py:24]: Typo

File: api/ellandi/registration/serializers.py [hunk 2]
```diff
@@ -19,6 +19,27 @@
 )
 
 
+class ChoiceDisplayField(serializers.ChoiceField):
```
Comment (Singletoned) [inline api/ellandi/registration/serializers.py:22]: You could just change the choices to not use an internal value.  EG:
```
    class SkillLevel(models.TextChoices):
        BEGINNER = ("Beginner", "Beginner")
        ADVANCED_BEGINNER = ("Advanced beginner", "Advanced beginner")
        COMPETENT = ("Competent", "Competent")
        PROFICIENT = ("Proficient", "Proficient")
        EXPERT = ("Expert", "Expert")
```

Comment (nmenezes0) [reply api/ellandi/registration/serializers.py:22]: That is probably simpler!

File: api/ellandi/registration/serializers.py (hunk 1 above)
Comment (nmenezes0) [reply api/ellandi/registration/serializers.py]: Now deleted.

File: api/ellandi/registration/serializers.py (hunk 2 above)
Comment (nmenezes0) [reply api/ellandi/registration/serializers.py]: Have changed.

**Comment by nmenezes0** [comment]
@rottitime 

The field names now look like this:
```
  "skills": [
    {
      "name": "string",
      "level": "Beginner",
      "validated": true
    }
  ],
  "languages": [
    {
      "name": "string",
      "speaking_level": "Basic",
      "writing_level": "Basic"
    }
  ],
  "skills_develop": [
    {
      "name": "string"
    }
  ]
  ```

**Review by Singletoned — APPROVED** [review]

**Pull request merged**